igcollect/linux_disklatency.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+"""igcollect - Linux Disk I/O latency
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import libvirt, sys
+
+def main():
+    URI = "qemu:///system"
+    strpoolname = 'xen-data'
+    strvolname = 'test-vol'
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='None')
+    args = parser.parse_args()
+    template = args.prefix + '.{}.{} {} ' + str(int(time()))
+
+    try:
+        conn = libvirt.open(URI)
+    except:
```

Comment:
Catching all possible exceptions is a bad idea, you're are interested only in libvirt exceptions.

Reply:
changed